static/scripts/app-register.js

```diff
@@ -1,7 +1,55 @@
 const formbtn = document.querySelector('.btn');
+const formInputs = document.querySelectorAll('.form-inputs');
 // http://localhost:2228/register
 // const inputs = document.querySelectorAll('.form-inputs');
 
+function keyDownEvents(){
+    const REGEXP = regExpDelivery();
+
+    for(let i = 0; i < formInputs.length; i++){
+
+        formInputs[i].addEventListener('input', () => {
+            
+            let user_err = document.createElement('small');
+            if(!REGEXP[i].test(formInputs[i].value)){
+                // console.log(formInputs[i].nextElementSibling.nodeName)
+                if(formInputs[i].nextElementSibling?.classList.contains('user-err')){
+                    formInputs[i].nextElementSibling.remove();
+                }
+                user_err.classList.add('user-err');
+                user_err.innerText = keydownMessages(formInputs[i])[i];
+                formInputs[i].style.border = '2px solid #b90909'
+                formInputs[i].after(user_err);
+                formbtn.disabled = true
+                
+            } else if(REGEXP[i].test(formInputs[i].value)){
+                if(formInputs[i].nextElementSibling?.classList.contains('user-err')){
```

Comment:
We do this within both paths of if and else, why not move it outside?

Reply:
![quickfix](https://github.com/FirstDayAtWork/spfy-app-pages/assets/100528867/433432d9-d198-4c24-a547-8cabeee5ef59)

fixed

> We do this within both paths of if and else, why not move it outside?

Reply:
good, please commit it and I will close the comment :)